.travis.yml

```diff
@@ -2,6 +2,7 @@ language: java
 before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
+  - "chmod +x gradlew"
```

Comment:
This should not be necessary.  If you execute `chmod +x gradlew` on your local machine then commit it, Git will remember the permissions.

Reply:
Finished